src/celengine/framebuffer.h

```diff
@@ -15,20 +15,22 @@
 class FramebufferObject
 {
  public:
-    enum
+    enum class AttachmentType
     {
-        ColorAttachment = 0x1,
-        DepthAttachment = 0x2
+        None,
```

Comment:
Better add new bit fields

Reply:
what do you mean here? AttachmentType cannot be a union of multiple.